client/src/lib/components/CaptchaV2.svelte

```diff
@@ -52,16 +75,25 @@
 
 <svelte:head>
   {#if componentMounted}
```

Comment:
I don't know if this line is needed, tho this was before your changes so you can ignore it.